scripts/forge_pod_template.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
```

Comment:
Instead of storing the template here, the template could be a BatchJob created by the TF/Helm (like the validator backup restore job) or a CronJob (with `suspend=true`). The script would then `kubectl get` the template.

Reply:
I think the main purpose of this pod template is used for deploy forge pod on cluster to run test itself. We might can split this functional part for template into another Job as mentioned above

Reply:
@mgorven yea I thought of that approach as well. Editing and applying the template would then need some python scripting magic like we do for restore. I think in the future we can probably go that route if needed (along with python rewrite and s3 locking), but this works for now. A singular pod's lifecycle is easy to reason about.